mass: 5.0 # mass of the vehicle
max_force: 5.0 # maximum force the motor can apply to the vehicle
max_speed: 2.0 # maximum speed of the vehicle


# TOPICS FOR TURTLESIM"
vel_topic_sub: "/vel"
vel_topic_pub: "/game_navigation/unsafe/cmd_vel"
pose_topic: "/turtle1/my_pose"

# TOPICS FOR TRISKAR
# vel_topic: "/turtle1/vel"
# pose_topic: "/turtle1/my_pose"